right_arm:
	tip_name:right_wrist
	root_name:right_arm_mount
	kinematics_solver: arm_kinematics_constraint_aware/KDLArmKinematicsPlugin
left_arm:
	tip_name:left_wrist
	root_name:left_arm_mount
	kinematics_solver: arm_kinematics_constraint_aware/KDLArmKinematicsPlugin

## Modify the conditions to include the sensor data for collision checking
use_collision_map: false
collision_map_safety_timeout: 10000.0
joint_states_safety_timeout: 1.0
tf_safety_timeout: 10.0
default_robot_padding: 0.02
link_padding:
  - link: right_arm 
    padding: .02
  - link: left_arm 
    padding: .02
